docs/atlascli/command/atlas-dataLakes-create.txt

```diff
@@ -37,7 +39,7 @@ Arguments
    * - name
      - string
      - true
-     - Name of the Atlas Data Lake to create.
+     - Name of the Atlas federated database to create.
```

Comment:
We should refer to the product as Atlas Data Federation and the virtual databases as federated database instances. So, suggest:
```suggestion
     - Name of the federated database instance to create.
```